docs/en/pact-reference.md

```diff
@@ -1530,42 +1530,22 @@ To share new values
 
-Pacts are designed to run in one of two different contexts, private and public. A private pact is
-indicated by each step identifying a single entity to execute the step, while public steps do
-not have entity indicators. A pact can only be uniformly public or private: if some steps
-has entity indicators and others do not, this results in an error at load time.
-
-### Public Pacts
-Public pacts are comprised of steps that can only execute in strict sequence. Any enforcement of who can execute a step
+Pacts are comprised of steps that can only execute in strict sequence. Any enforcement of who can execute a step
 happens within the code of the step expression. All steps are "manually" initiated by some participant
 in the transaction with CONTINUATION commands sent into the blockchain.
 
-### Private Pacts
-Private pacts are comprised of steps that execute in sequence where each step only executes on entity
-nodes as selected by the provided 'entity' argument in the step; other entity nodes "skip" the step.
-Private pacts are executed automatically by the blockchain platform after the initial step is sent
-in, with the executing entity's node automatically sending the CONTINUATION command for the next step.
-
 ### Failures, Rollbacks and Cancels
 
-Failure handling is dramatically different in public and private pacts.
-
-In public pacts, a rollback expression is specified to indicate that the pact can be "cancelled" at
+In pacts, a rollback expression is specified to indicate that the pact can be "cancelled" at
 this step with a participant sending in a CANCEL message before the next step is executed. Once the last
 step of a pact has been executed, the pact will be finished and cannot be rolled back. Failures
 in public steps are no different than a failure in a non-pact transaction: all changes are rolled back.
 Pacts can therefore only be canceled explicitly and should be modeled to offer all necessary cancel options.
 
-In private pacts, the sequential execution of steps is automated by the blockchain platform itself. A failure
-results in a ROLLBACK message being sent from the executing entity node which will trigger any rollback expression
-specified in the previous step, to be executed by that step's entity. This failure will then "cascade" to the
-previous step as a new ROLLBACK transaction, completing when the first step is rolled back.
-
 ### Yield and Resume
 
-A step can yield values to the following step using [yield](pact-functions.html#yield) and [resume](pact-functions.html#resume). In public,
-this is an unforgeable value, as it is maintained within the blockchain pact scope. In private, this is
-simply a value sent with a RESUME message from the executed entity.
+A step can yield values to the following step using [yield](pact-functions.html#yield) and [resume](pact-functions.html#resume).
+This is an unforgeable value, as it is maintained within the blockchain pact scope.
```

Comment:
Relevant aside: since Pact is moving towards being marketed as a backend-agnostic, is it still safe to assume that `resume` values are unforgeable for any backend?

Reply:
For now, that change should come at a later time.